src/routes/landing/containers/__tests__/LandingPage.spec.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { shallow, mount } from 'enzyme'
+import { Provider } from 'react-redux'
+import ConnectedLandingPage, { LandingPage } from '../LandingPage'
+import Store from 'Store'
+import Payload from 'modules/endpoint/Payload'
+import LocationModel from '../../../../modules/endpoint/models/LocationModel'
+import { mockLocations } from 'setupMocks'
+
+describe('LandingPage', () => {
+  const locations = [
+    new LocationModel({
+      name: 'City',
+      code: 'city',
+      live: true,
+      eventsEnabled: false,
+      extrasEnabled: false
+    })
+  ]
+
+  test('should match snapshot', () => {
+    expect(shallow(<LandingPage locations={locations} language={'de'} />)).toMatchSnapshot()
+  })
+
+  describe('connect', () => {
+    test('should map state to props', () => {
+      const language = 'en'
+
+      const store = new Store()
+      store.init({
+        languages: new Payload(false),
+        router: {params: {language}}
+      })
+
+      const tree = mount(
+        <Provider store={store.redux}>
+          <ConnectedLandingPage />
+        </Provider>
+      )
+
+      const landingPageProps = tree.find(LandingPage).props()
+
+      expect(landingPageProps).toEqual({
```

Comment:
you could also do a snapshot test here

Reply:
but I think it's better like this

Reply:
yeah not sure. I won't touch it again, but we need a real reason when to use snapshots and when just a deep equal :/

Reply:
jop